src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -59,6 +60,9 @@ public Climber() {
         // Zero encoder (assumed zero at startup)
         rightEncoder.setPosition(0);
         leftEncoder.setPosition(0);
+
+        leftMotor.setSoftLimit(CANSparkBase.SoftLimitDirection.kReverse, 0);
```

Comment:
This looks correct, but we'll need to verify it.  Evan indicated that it might not have worked (possibly due to motor inversion issues?)